site-src/geps/gep-713.md

```diff
@@ -0,0 +1,598 @@
+  policy API implementations
+* Enable policy attachment at all relevant scopes, including Gateways, Routes,
+  Backends
+* Ensure the policy attachment specification is generic and forward thinking
+  enough that it could be easily adapted to other grouping mechanisms like
+  Namespaces in the future
+* Provide a means of attachment that works for both ingress and mesh
+  implementations of this API
+* Provide a consistent specification that will ensure familiarity between both
+  included and implementation-specific policies so they can both be interpreted
+  the same way.
+
+## Out of scope
+
+* Define all potential policies that may be attached to resources
+* Design the full structure and configuration of policies
+
+## API
+
+This approach is building on concepts from all of the alternatives discussed
+below. This is very similar to the existing BackendPolicy resource in the API,
+but also borrows some concepts from the [ServicePolicy
+proposal](https://github.com/kubernetes-sigs/gateway-api/issues/611).
+
+### Policy Attachment for Ingress
+Attaching policy to Gateway resources for ingress use cases is relatively
+straightforward. A policy can reference the resource it wants to apply to.
+Access is granted with RBAC - anyone that has access to create a RetryPolicy in
+a given namespace can attach it to any resource within that namespace.
+
+![Simple Ingress Example](images/713-ingress-simple.png)
+
+To build on that example, it’s possible to attach policies to more resources.
+Each policy applies to the referenced resource and everything below it in terms
+of hierarchy. Although this example is likely more complex than many real world
+use cases, it helps demonstrate how policy attachment can work across
+namespaces.
+
+![Complex Ingress Example](images/713-ingress-complex.png)
+
+### Policy Attachment for Mesh
+Although there is a great deal of overlap between ingress and mesh use cases,
+mesh enables more complex policy attachment scenarios. For example, you may want
+to apply policy to requests from a specific namespace to a backend in another
+namespace.
+
+![Simple Mesh Example](images/713-mesh-simple.png)
+
+Policy attachment can be quite simple with mesh. Policy can be applied to any
+resource in any namespace but it can only apply to requests from the same
+namespace if the target is in a different namespace.
+
+At the other extreme, policy can be used to apply to requests from a specific
+workload to a backend in another namespace. A route can be used to intercept
+these requests and split them between different backends (foo-a and foo-b in
+this case).
+
+![Complex Mesh Example](images/713-mesh-complex.png)
+
+### Policy TargetRef API
+
+Each Policy resource should include a single `targetRef` field. It must not
+target more than one resource at a time, but it can be used to target larger
+resources such as Gateways or Namespaces that may apply to multiple child
+resources.
```

Comment:
```suggestion
Each Policy resource MUST include a single `targetRef` field. It MUST NOT
target more than one resource at a time, but it MAY be used to target larger
resources such as Gateways or Namespaces that may apply to multiple child
resources.
```

Reply:
Updated, thanks!